src/main/java/aisu/command/AddCommand.java

```diff
@@ -0,0 +1,62 @@
+package aisu.command;
+
+import aisu.exception.AisuException;
+import aisu.storage.Storage;
+import aisu.tasklist.TaskList;
+import aisu.ui.Ui;
+import aisu.task.Task;
```

Comment:
It would be good to standardise imports lexicographically (e.g. aisu.a*** to aisu.z***)
```suggestion
import aisu.exception.AisuException;
import aisu.storage.Storage;
import aisu.task.Task;
import aisu.tasklist.TaskList;
import aisu.ui.Ui;
```